PocketKit/Sources/PocketKit/Home/CompactHomeCoordinator.swift

```diff
@@ -93,6 +89,40 @@ class CompactHomeCoordinator: NSObject {
             show(viewModel)
         case .recommendation(let viewModel):
             show(viewModel)
+        case .webViewRecommendation(let viewModel):
+            viewModel.$presentedAlert.sink { [weak self] alert in
+                self?.present(alert: alert)
+            }.store(in: &readerSubscriptions)
+
+            viewModel.$selectedRecommendationToReport.sink { [weak self] recommendation in
+                self?.report(recommendation)
+            }.store(in: &readerSubscriptions)
+
+            viewModel.events.sink { [weak self] event in
+                switch event {
+                case .contentUpdated:
+                    break
+                case .archive, .delete:
+                    self?.popToPreviousScreen()
+                }
+            }.store(in: &readerSubscriptions)
+
+            present(url: viewModel.url)
```

Comment:
Can we bring this into its own function like we do in some cases above (such as `.recommendation`)?